src/expression.h

```diff
@@ -112,6 +113,7 @@ class Range : public Expression
 	ValuePtr evaluate(const class Context *context) const;
 	virtual void print(std::ostream &stream) const;
 private:
+    virtual bool isLiteral() const { return true;}
```

Comment:
@amarjeetkapoor1 This also looks a bit weird. Ranges are only literals if all its elements are literal, same as for Vector.